tests/fail/tokio/sleep.stderr

```diff
@@ -0,0 +1,68 @@
+error: unsupported operation: returning ready events from epoll_wait is not yet implemented
+  --> CARGO_REGISTRY/.../epoll.rs:LL:CC
+   |
+LL | /         syscall!(epoll_wait(
+LL | |             self.ep,
+LL | |             events.as_mut_ptr(),
+LL | |             events.capacity() as i32,
+LL | |             timeout,
+LL | |         ))
+   | |__________^ returning ready events from epoll_wait is not yet implemented
+   |
+   = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
+   = note: BACKTRACE:
+   = note: inside `mio::sys::PLATFORM::selector::epoll::Selector::select` at CARGO_REGISTRY/.../mod.rs:LL:CC
```

Comment:
Please filter out allthe notes. While we only run these tests on linux hosts,and thus don't suffer from the usual platform differences that other backtraces have, they convey no useful information for us to track. There should be examples of existing filters that you can copy here  if you grep for `//@.*= note`

Reply:
I didn't find this, but I got those lines filtered out. Let me know if it should be done differently.